docs/hardware.rst

```diff
@@ -263,62 +263,41 @@ Specific Hardware Recommendations
+This optional support is preferable in whichever solution you choose for your
+servers, since you want neither Wifi nor Bluetooth; if you're repurposing old
+hardware, be sure to remove these components if they exist.
 
-.. note:: If you purchase the NUC from `Amazon
-	  <http://www.amazon.com/Intel-D54250WYK-DisplayPort-Graphics-i5-4250U/dp/B00F3F38O2/>`__,
-	  make sure you click "With Powercord" to have one included in
-	  the package.
-
-The NUCs come as kits, and some assembly is required. You will
-need to purchase the RAM and hard drive separately for each NUC and
-insert the cards into the NUC before it can be used. We recommend:
-
--  2 `240 GB SSDs <http://www.amazon.com/dp/B00BQ8RKT4/>`__
--  A `4 GB (4GBx2) memory
-   kit <http://www.amazon.com/Crucial-PC3-12800-204-Pin-Notebook-CT2CP25664BF160B/dp/B005MWQ6WC/>`__
+The NUCs come as kits, and some assembly is required. You will need to purchase
+the RAM and hard drive separately for each NUC and insert the cards into the NUC
+before it can be used. We recommend:
 
+-  2x 240 GB SSDs
+-  1x 4 GB (4GBx2) memory kit
    -  You can put one 4GB memory stick in each of the servers.
 
-.. note:: The D54250WYK has recently been `EOL'ed by Intel
-	  <http://ark.intel.com/products/series/70407/Intel-NUC-Boards-and-Kits>`__.
-	  Availability and prices may be subject to change. We are
-	  working on analyzing alternative recommendations, but there
-	  are no immediately obvious alternatives that share the
-	  benefits of the D54250WYK (primarily, the lack of integrated
-	  wireless networking hardware).
-
-.. note:: An earlier release of SecureDrop (0.2.1) was based on Ubuntu
-	  12.04.1 (precise). We encountered issues installing this
-	  version of Ubuntu on some types of Intel NUCs. The problem
-	  manifested after installing Ubuntu on the NUC. The
-	  installation would complete, but rebooting after
-	  installation would not succeed.
-
-	  We have not encountered this or any similar problems in
-	  testing the current release series (0.3.x) with the Intel
-	  NUCs. Since 0.3 is based on Ubuntu 14.04.1 (trusty), we
-	  believe the issue has been resolved in the newer release of
-	  Ubuntu.
-
-	  If you do encounter issues booting Ubuntu on the NUCs, try
+.. note:: The D54250WYK we previously recommended has now entered `End of Life`
+    and `End of Interactive Support` statuses. If you're currently using this
+    model for your SecureDrop setup, and need hardware support, you'll need to
+    consult the `support community <https://communities.intel.com/community/tech/nuc>`__ forum.
+
+.. note:: If you encounter issues booting Ubuntu on the NUCs, try
 	  updating the BIOS according to `these instructions
 	  <http://arstechnica.com/gadgets/2014/02/new-intel-nuc-bios-update-fixes-steamos-other-linux-booting-problems/>`__.
 
-.. note:: Some BIOS versions on the NUC will cause the server to
+.. caution:: Some BIOS versions on the NUC will cause the server to
```

Comment:
The thread below is marked **Answered** and one of the [latest comments](https://communities.intel.com/message/432692#432692) dated november 2016 point to [the release notes of the BIOS version fixing the issue](https://downloadmirror.intel.com/26263/eng/RY_0359_ReleaseNotes.pdf). Although the associated comment is terse (**Fixed an issue where the NUC can't wake up from S3 by pressing the power button in Linux.**), it looks this warning can be removed.

Reply:
Excellent! Thanks for the update on that issue.